code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -331,3 +331,20 @@
 	icon_dead = "penny_dead"
 	icon_rest = "penny_rest"
 	holder_type = /obj/item/holder/cat/penny
+/mob/living/simple_animal/cat/crusher // Goodbye Runtime, make way for the new girl in town.
```

Comment:
Missing new line after the previous block

Reply:
When I tried to get the sprite added along with map changes, Developers informed me that sprites and map changes have to be seperate. Ping offered to PR the sprites for me while I was/am busy finishing finals so that I can then map it in this upcoming weekend.